### ISSUE TITLE
Nice project! Is there a way to speed up the text showing speed as in ChatGPT plus?

### ISSUE THREAD

**Issue by xc-mezcal** [danny-avila/chatgpt-clone#185]
Hello,

I tried this clone and it works amazing. The output result is a bit different than what I got from the website direclty, but it's OK!

One thing really annoy me is when chatgpt showing the answer --- the text showing speed is too slow compared what I got from the chatgpt-plus using gpt-3.5 model. It will take too long to completely show a long answer.

If the api call response all needed information in one time, can we speed up the text showing procedure in the broswer?  

Any hints on how to speed up the text showing procedure? Many thanks!

**Comment by danny-avila** [comment]
Hey, thanks for checking out the project!

Can you let me know what endpoint you’re using (what you have selected by clicking the icon in text input)? 

Assuming you are using the OpenAI endpoint, that is a call to the official completions API speed is totally dependent on OpenAI. I’ve noticed this varies and can sometimes be really fast or otherwise slower. 

It would be more fair to compare to the Browser endpoint which is using the same backend API as the browser/official site. I haven’t used 3.5 much with the browser endpoint since I only use gpt 4 with that. Even then, you are at the mercy of OpenAI.

In short, the streaming speed is not something I’ve configured and is totally dependent on third party (OpenAI)

**Comment by xc-mezcal** [comment]
Yep. It's openai chatgpt-3.5-turbo api. Thanks for noticing me that the text showing speed is controled by OpenAI.
